packages/cxl-ui/scss/cxl-card-grid.scss

```diff
@@ -0,0 +1,28 @@
+@use "~@conversionxl/cxl-lumo-styles/scss/mq";
+
+:host {
+  position: relative;
+  display: block;
+
+  ::slotted(.grid) {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    gap: var(--lumo-space-m);
```

Comment:
Why the need to slto a .grid element? If we style the host to be the grid, we achieve the same result and allow parent element to adapt the styling if needed. That's important in our setup, to be able to easily re-use the element in wp and make changes by applying desired classes. Just slotted the cards directly.